README.md

```diff
@@ -8,19 +8,8 @@ LINZ uses [Argo workflows](https://argoproj.github.io/workflows/) for running bu
 
 ## Command Line Tools
 
-- [lds-fetch-layer](#lds-fetch-layer)
-- [create-manifest](#create-manifest)
-- [copy](#copy)
-- [group](#group)
-- [generate-path](#generate-path)
-- [list](#list)
-- [pretty-print](#pretty-print)
-- [stac catalog](#stac-catalog)
-- [stac github-import](#stac-github-import)
-- [stac sync](#stac-sync)
-- [stac validate](#stac-validate)
-- [tileindex-validate](#tileindex-validate)
-- [bm-create-pr](#bm-create-pr)
+List of [Commands](./COMMANDS.md) and their usage
```

Comment:
Suggest to make the link the whole sentence:

```suggestion
[Commands and their usage](./COMMANDS.md)
```

By the way, does that file make most of this file redundant?

Reply:
> By the way, does that file make most of this file redundant?

Soon :D I need to add the "examples" into the commands then we can drop the rest of the README.